### PR TITLE
Update: BDP-2906: mitigate node oversubscription

### PR DESCRIPTION
**Pull request by jjmiller-vs**

### CODE REVIEW

File: nhd/K8SMgr.py [hunk 1]
```diff
@@ -111,53 +113,53 @@ def GetPodNode(self, pod, ns):
...
+            if ret is None:
                 return ''
 
             return ret.spec.node_name
-            
+
         except ApiException as e:
-            self.logger.error("Exception when calling CoreV1Api->read_namespaced_pod: %s\n" % e)                  
-        
+            self.logger.error(f"Exception when calling CoreV1Api->read_namespaced_pod: %s\n{e}")
```
Comment (urbanatb) [inline nhd/K8SMgr.py:124]: Remove %s

File: nhd/K8SMgr.py [hunk 2]
```diff
@@ -111,53 +113,53 @@ def GetPodNode(self, pod, ns):
...
 
     def GetPodObj(self, pod, ns):
-        try: 
+        try:
             pobj = self.v1.read_namespaced_pod(pod, ns)
             return pobj
         except ApiException as e:
-            self.logger.error("Exception when calling CoreV1Api->read_namespaced_pod: %s\n" % e)  
-        
+            self.logger.error("Exception when calling CoreV1Api->read_namespaced_pod: %s\n{e}")
```
Comment (urbanatb) [inline nhd/K8SMgr.py:133]: Missing f - remove %s

**Review by urbanatb — APPROVED** [review]

File: nhd/K8SMgr.py (hunk 1 above)
Comment (jjmiller-vs) [reply nhd/K8SMgr.py]: > Remove %s

Nice catch - thank-you

File: nhd/K8SMgr.py (hunk 2 above)
Comment (jjmiller-vs) [reply nhd/K8SMgr.py]: > Missing f - remove %s

Another nice catch - thank-you

**Pull request merged**